### PR TITLE
White VOD chat username theatre mode

### PR DESCRIPTION
**Pull request by sparcut**
Fixes problem in VOD chat replay (theatre mode) where usernames are white instead of colour.

No good:
chrome_2017-10-06_19-56-53

### CODE REVIEW

**Comment by sparcut** [comment]
`better fix` commit readds original style.

Instead adds custom rule for
```css
.video-chat__message-author span { ... }
```

File: src/css/betterttv-dark.css [hunk 1]
```diff
@@ -3807,6 +3807,10 @@ button.follow-button, .subscription-modal__right button, .subscription-modal__le
     color: #d3d3d3;
 }
 
+.video-chat__message-author span {
+	color: inherit;
```
Comment (night) [inline src/css/betterttv-dark.css]: we use spaces not tabs

Comment (sparcut) [reply src/css/betterttv-dark.css]: That's strange, my editor should not be set to tabs... I use spaces.

Will fix now.

**Comment by sparcut** [comment]
@night Fixed tab and editor also found a few semantic errors which I included in the second commit.

**Pull request merged**